src/test/test_stats.c

```diff
@@ -245,6 +246,43 @@ test_rephist_mtbf(void *arg)
   tor_free(ddir_fname);
 }
 
+static void
+test_commit_max(void *arg)
```

Comment:
I don't know if this a good way to test this function because
I am manipulating the data structure directly which is implementation dependent.
Is this ok ???

Reply:
Yes, that's fine.